dask/core.py

```diff
@@ -287,6 +287,7 @@ def subs(task, key, val):
     >>> subs((inc, 'x'), 'x', 1)  # doctest: +SKIP
     (inc, 1)
     """
+    hash_key = {key}
```

Comment:
Teeny tiny nit: I would probably move `hash_key` definition down 10 lines or so to right before the `for` loop.  Fine as is though.

Reply:
Fixed